src/Compilers/Core/Portable/FileSystem/PathUtilities.cs

```diff
@@ -732,6 +732,10 @@ public static bool IsValidFilePath([NotNullWhen(true)] string? fullPath)
         /// If the current environment uses the '\' directory separator, replaces all uses of '\'
         /// in the given string with '/'. Otherwise, returns the string.
         /// </summary>
+        /// <remarks>
+        /// This method is equivalent to Microsoft.CodeAnalysis.BuildTasks.GenerateMSBuildAnalyzerConfig.NormalizeWithForwardSlash
```

Comment:
>Microsoft.CodeAnalysis.BuildTasks.GenerateMSBuildAnalyzerConfig.NormalizeWithForwardSlash [](start = 41, length = 89)

nit: consider making it a link (`<see` I think). Same on other end

Reply:
You can't sadly, because they don't reference each other (why we have to copy). It issues a warning that it can't be resolved.

---
In reply to: [419837629](https://github.com/dotnet/roslyn/pull/43617#discussion_r419837629) [](ancestors = 419837629)